armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -74,13 +74,21 @@ def expandColdDimsToHot(
     referenceAssembly: :py:class:`Assembly <armi.reactor.assemblies.Assembly>`, optional
         Assembly whose mesh other meshes will conform to if isDetailedAxialExpansion is False.
         If not provided, will assume the finest mesh assembly which is typically fuel.
+
+    Notes
+    -----
+    Calling this method will result in an increase in mass via applyColdHeightMassIncease!
```

Comment:
Typo in increase

Reply:
Addressed https://github.com/terrapower/armi/pull/1342/commits/00808d2e8f513d95fbdd3d48fdad495eaaea3ce0